### DISCUSSION TITLE
Can not install / update search plugins

### DISCUSSION THREAD

**Discussion by DirtyMarx** [qbittorrent/search-plugins#235] [Q&A]
I just did a fresh install and cannot install any plugins
![qbittorrent_CvijD5JCjM](https://user-images.githubusercontent.com/101524909/206091337-8dcd8d54-a14e-4800-998e-38138f36f413.png)
![qbittorrent_NoqXlEFddm](https://user-images.githubusercontent.com/101524909/206091412-675ff053-f5b0-4faa-916e-2e3097b3fe65.png)

**Comment by Melodey757** [comment]
I'm having the same problem... Search Plugins were there yesterday, gone today.. re-installing didn't work..

**Comment by sausje** [comment]
Yep, updated and wanted to try out search for the first time... Welp, it doesn't work, and it seems for a month already... Bad...

**Comment by p060477** [comment]
me too....this is a very big BUG and the silence from dev is so astonishing...what a shame... :(

**Comment by Protey** [comment]
I confirm the problem is still not solved!!!

**Comment by ngosang** [comment]
How to fix:

1. Close qBittorrent from the Tray or Task manager.
2. Remove the folder `nova3` => https://github.com/qbittorrent/search-plugins/wiki/How-to-configure-Jackett-plugin#configuration-file
3. Start qBittorrent
4. Go to Search plugins, in the modal dialog click on "Check for updates". All the official plugins will be installed again

NOTE: If you have unofficial plugins you will have to install them again.

**Comment by LaFaZion** [comment]
ngosang

I occur the same problem as well, would you please specify which folder needed to be removed you were talking about ? I can't find any instructions on that page you mentioned. Thanks.

**Reply by Naradon64** [reply]
read the guide carefully, it's in there.
go to appdata>local and find qbittorent folder. nova3 is in there

**Comment by LaFaZion** [comment]
Naradon64

Thanks man, I found and deleted the nova3 folder but all the official plug-in are still "not supported".........

**Reply by Naradon64** [reply]
I try to downgrade from the version 5.0.0 to 4.x.x by deleted the program and install the previous version which can be found on the official website (click on download and it will take you to another page that you can select the version to download)
do this and delete nova3 again and see if it helps. 
this worked for me.

**Reply by LaFaZion** [reply]
I did your way to downgrade to ver. 4.6.7 and all search engines are back ! I think this is the most easiest way to deal with this issue.
Thanks again man !!

**Comment by xavier2k6** [comment]
@LaFaZion See -> https://github.com/qbittorrent/qBittorrent/issues/21579#issuecomment-2406910742

**Reply by LaFaZion** [reply]
Thank you so much for your help xavier2k6, and that's very technical and detailed, but I was not sure I can do it correctly so the most easiest way for me is downgrade the torrent client to older version and it works for me, search engines are back !
Thanks again, have a nice day man !
